src/patternfly/components/Spinner/spinner.scss

```diff
@@ -1,32 +1,47 @@
 // Spinner Component
 
 .pf-c-spinner {
-  --pf-c-spinner--AnimationDuration: 1.5s;
-  --pf-c-spinner--AnimationTimingFunction: cubic-bezier(.77, .005, .315, 1);
-
-  // This conceptual variable is used to set other values all based on the diameter of the spinner.
-  // The default is the -xl modifier size.
   --pf-c-spinner--diameter: var(--pf-global--icon--FontSize--xl);
-
-  // These conceptual variables are used to size the stroke width, which is accomplished using various properties in the clipper:after and the ball animations
-  --pf-c-spinner--stroke-width-multiplier: .1;
-  --pf-c-spinner--stroke-width: calc(var(--pf-c-spinner--diameter) * var(--pf-c-spinner--stroke-width-multiplier));
-
-  // Set the overall width and height of the spinner to be consistent with the icon size
-  // Width and Height must be the same
   --pf-c-spinner--Width: var(--pf-c-spinner--diameter);
   --pf-c-spinner--Height: var(--pf-c-spinner--diameter);
-
-  // This variable is used to set the color for all the parts of the spinner
   --pf-c-spinner--Color: var(--pf-global--primary-color--100);
 
-  // Modifier variables for main spinner
-  // Width and Height must be the same
+  // sizes
   --pf-c-spinner--m-sm--diameter: var(--pf-global--icon--FontSize--sm);
   --pf-c-spinner--m-md--diameter: var(--pf-global--icon--FontSize--md);
   --pf-c-spinner--m-lg--diameter: var(--pf-global--icon--FontSize--lg);
   --pf-c-spinner--m-xl--diameter: var(--pf-global--icon--FontSize--xl);
 
+  width: var(--pf-c-spinner--Width);
+  height: var(--pf-c-spinner--Height);
```

Comment:
You also have width/height declared under `span.pf-c-spinner` set to the same values - do you need them both?

Reply:
Nope! Updated.